### ISSUE TITLE
Upgrade version of Bootstrap used in example projects to v3.x

### ISSUE THREAD

**Issue by jonbullock** [jbake-org/jbake#210]
Upgrade version of Bootstrap used in example projects to v3.x

**Comment by msgilligan** [comment]
Is the process for upgrading Bootstrap version documented?

I'd love to see a way to rebuild the minified Bootstrap files into the `jbake/assets` folder. Any suggestions on how to do that? Something using the Asset Pipeline Gradle Plugin would be ideal for the use cases I have in mind.

**Comment by jonbullock** [comment]
No it's not... it's been a manual process so far, there was some CSS conflicts between Asciidoctor & Bootstrap that I had to manually resolve. My plan was to create a custom backend for Asciidoctor so that we could add a CSS prefix to avoid any conflicts.

Processing of assets is something I'd like to add... something I was hoping the extension/plugin system would help enable.

**Comment by ieugen** [comment]
I had a lot of time thinking about how to build some nice tooling for static websites and I think Gradle is a good candidate for this kind of stuff. It is a build system is deigned to build stuff so it is a good candidate for packaging and compiling files (sources and assets). 

Because it has a standard (more or less) build file you can design recipes to compile SASS or transform images and so on. You can also leverage the entire plugin ecosystem. 

Of course, care must be taken on the integration part so it doesn't stand in the way.

**Comment by jonbullock** [comment]
Yeah I like the idea of using a build system to produce example templates and deal with the compilation of Bootstrap.
